CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## Unreleased
+ * `ERC2981`: make `royaltiInfo` public to allow super call in overrides. ([#3305](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/3305))
```

Comment:
Nothing major, but `royaltiInfo` should be corrected to `royaltyInfo`

Reply:
Thanks for noticing! Well update it.